README.md

```diff
@@ -1,436 +1,95 @@
 
-Consider the `metrilo.load("<YOUR_PROJECT_TOKEN>")` line. Here you should put your project token. You can find it on the Settings page in the Metrilo web-site.
+### Implementation
+For your convenience we have made a javascript library that will simplify calling the [Frontend endpoints](#frontend-endpoints). You could load it on your website and use the provided functions with it. [Here](frontend_integration_library.md) you could check how to use the library.
 
-#### Identify user ####
+#### Frontend endpoints
+* /article/view
+* /category/view
+* /product/view
+* /cart/add
+* /cart/remove
+* /checkout
+* /custom
+* /page/land
+* /page/view
+* /search
+* /visitor/apply-tags
+* /visitor/identify
 
-Metrilo tracks anonymous user history and it is nice to notify Metrilo when you have customer details. The common use cases where identification could be done are:
-* After order, because each order contains customer details
-* After subscription for some resource on your web-site
-* After login
+### Examples
+Customer actions and what you have to do with the loaded library(the params are not included):
+* visits the Homepage -> call metrilo.viewPage
+* visits an article page -> call metrilo.viewArticle
+* visits a category page -> call metrilo.viewCategory
+* visits a product page -> call metrilo.viewProduct
+* visits another page(e.g. /thank-you) -> call metrilo.viewPage
+* types something in the search bar -> call metrilo.search
+* adds a product to his cart -> call metrilo.addToCart
+* removes a product to his cart -> call metrilo.removeFromCart
+* checks the content of his cart -> call metrilo.viewPage
+* goes to the checkout page -> call metrilo.checkout
+* enters his email in some html form -> call metrilo.identify
 
-The following example shows how identification works:
-```javascript
-var identifyParams = { name: 'Johny Bravo', first_name: 'Johny', last_name: 'Bravo', email: 'johnybravo@gmail.com' };
+### About the identify
+When a customer comes to the store and he is not loged in Metrilo treats him as an anonymous visitor. Metrilo adds a unique identifier to his session, so it could attribute his actions to this identifier. Once he provides his email(e.g. logs in, signs up, fills in a subscription form, makes an order, etc.) you have to call ```metrilo.identify(customerEmail)```. This will associate the unique identifier with the email. Moreover, all events that the visitor has while being an anonymous one will be merged with the events he has as identified(*if any). Keep in mind that the identify call must be done only once, the first time the visitor enters his email. If you are not sure that you have already called it, just check for the existence of email in the cookie named **cbuid**. If there is an email, no need to call it. This cookie contains either the unique identifier or the email and part of the identify function is to change the content  from the unique identifier to the email.
```

Comment:
I can make a link over there, but it seems better to me to stay here, because the file makes a  broad description about how everything works, not about how to make requests to the different endpoint or how to use the js library

Reply:
Actually, I will move it. It seems more reasonable when I read the tracking lib explanation